cylc/flow/task_events_mgr.py

```diff
@@ -477,8 +489,26 @@ def _process_message_check(
                 itask, itask.state, self.FLAG_RECEIVED_IGNORED, message,
                 event_time, submit_num, itask.submit_num)
             return False
-        if itask.state.status in (
-            TASK_STATUS_SUBMIT_RETRYING, TASK_STATUS_RETRYING
+
+        if (
+                itask.state(TASK_STATUS_WAITING)
+                and
+                (
+                    (
+                        # task has automatically submit-retried at least once
```

Comment:
```suggestion
                        # task has a submit-retry lined up
```
? ("has submit-retried at least one" suggests to me it has already RE-submitted at least once")